docs/src/devel/contributing.rst

```diff
@@ -164,6 +164,51 @@ data. A common pattern is validating the data file created by the
 associated export role, reading the data file and then calling the
 module you created for import.
 
+
+Commit Messages
+---------------
+
+For every pull request we request contributors to be compliant with the
+following notation to help generating automatically the project's changelog.
+
+Format
+^^^^^^
+
+.. code-block:: console
+
+    <feat>: <add an awesome feature>
+    ^----^  ^----------------------^
+    |       |
+    |       +-> Summary in present tense.
+    |
+    +-------> Type: [Nn]ew, [cC]hg, [fF]ix.
+
+    <body> ----> The commit's body.
+
+Accepted types:
+
+- `new` or `New`: newly implemented features
+- `chg` or `Chg`: changes in the CI automation, documentation or any other change not presented as a new feature
```

Comment:
I'd say CI automation should have a 'dev:' commit as it's not end-user facing. But this is just a nit. I think we can merge and tweak subsequently as we review docs.

Reply:
Cool we just need to adjust the changelog config when we update the docs so they are consistent, this patch covers only what we already have